homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -71,245 +79,225 @@
 )
 
 
-async def async_setup_platform(
```

Comment:
Not 100% sure, but I think the current process should be to add an import flow as well that will handle the current YAML structure. With a deprecation notice that shows when YAML support will be removed from core.

Does this PR add breaking changes in terms of that YAML support could not be imported due to different input required?

Reply:
See for an example how to import this: https://github.com/home-assistant/core/pull/64117.